ui/src/common/VerticalDivider/VerticalDivider.scss

```diff
@@ -1,4 +1,4 @@
 .__VerticalDivider__ {
-  background: $text-color;
-  margin-left: $padding-md - 1px;
+  background: $text-color !important;
+  margin-left: $padding-md - 1px !important;
```

Comment:
where this is helping? ☺️

Reply:
before it was working just by chance, but normally if you want to override styles with our self-written styles, on a ant component, it needs `!important`. Because it usually applies `class="class-name-from-prop antd-class ..."`